debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/transforms/ExtractNewDocumentState.java

```diff
@@ -153,6 +153,14 @@ public static ArrayEncoding parse(String value, String defaultValue) {
             .withDescription("Whether field names will be sanitized to Avro naming conventions")
             .withDefault(Boolean.FALSE);
 
+    public static final Field SANITIZE_VALUE = Field.create("sanitize.value")
```

Comment:
I see why you added the new option, but here's a suggestion: simply control this via the existing naming option above. If you want to use Avro, you also want to have this behavior for arrays, so I don't see a situtation where you'd configure the two settings differently. We always can add a dedicated option later on if really needed, but I want to fight option creep a bit. Thanks!

Reply:
I add a new one because the other one says : sanitize field name and not sanitize or sanitize.avro

Reply:
Yes, agreed a more generic field name would have been better. But I don't see any issue with re-using it for this purpose two. I.e. you'd never want to ensure Avro field names without also ensuring that proper Avro-compatible array behavior.